.github/workflows/manual-publish-docs.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  workflow_dispatch:
+
+name: Publish Documentation
+jobs:
+  build-publish:
+    runs-on: ubuntu-22.04
+    # Needed to get tokens during publishing.
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: erlef/setup-beam@v1
+        with:
+          version-type: loose
+          otp-version: 25.x
+          rebar3-version: 3.18.0
+      - uses: actions/checkout@v3
+      - id: build
+        name: Build Documentation
+        uses: ./.github/actions/build-docs
+
+      - uses: ./.github/actions/configure-rebar
```

Comment:
So the configure-rebar action is used wherever we need to publish. It both gets the token and puts it in a config file.